sources/iTermShellIntegrationWindowController.m

```diff
@@ -26,6 +26,8 @@
             return @"bash";
         case iTermShellIntegrationShellFish:
             return @"fish";
+        case iTermShellIntegrationShellFish:
```

Comment:
This should be Xonsh, not Fish. Make sure to test injection since it can be hard to get right.